analyzer/compliance-analyzer/src/main/java/eu/fasten/analyzer/complianceanalyzer/Main.java

```diff
@@ -0,0 +1,62 @@
+package eu.fasten.analyzer.complianceanalyzer;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.sql.SQLException;
```

Comment:
Unused import and in general imports are in the wrong order

Reply:
Solved in https://github.com/fasten-project/fasten/commit/7975246450da52793713310dc6123c14c876daf5.